src/components/searchSection/catergrayList.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+import { MDBIcon } from 'mdbreact';
+import './myStyle.scss';
+
+function CatergrayList({ items }) {
+  let itemsTags = [];
+  let numberOfItems = items.length;
+  let [ActiveBubble, setActiveBubble] = useState(0);
+  let [setCatogray] = useState('');
+
+  //light the bubble by pink color
+  const handelBubble = (index) => {
+    if (ActiveBubble === index) {
```

Comment:
variable name shouldn't start with upper-case letter 
```suggestion
    if (activeBubble === index) {
```